jparest/src/test/java/uk/gov/homeoffice/digital/sas/jparest/utils/PayloadCreator.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.homeoffice.digital.sas.jparest.utils;
+
+import java.util.List;
+
+public class PayloadCreator {
+
```

Comment:
Would also be good to privatise the constructor  as it's a utility class and we don't want it to be instantiated.
```suggestion
private PayloadCreator () {
        // no instantiation
    }
```

Reply:
Done